src/desktop/lib/preload/Electron.js

```diff
@@ -366,10 +366,10 @@ const Electron = {
         try {
             const content = await kdbx.exportVault(seeds, password);
             const now = new Date();
-
+            const prefix = seeds.length === 1 ? seeds[0].title : 'SeedVault';
```

Comment:
We should probably do a sanity check for special characters (eg `./\`), UTF-16 symbols (eg emojis) that could make the file name "troublesome" on some platforms. Maybe just leave ASCII symbols and filter out special characters?

Reply:
Great point, I'll address this in my next commit.

Reply:
@rajivshah3 and @rihardsgravis I couldn't find a better way to do it than duplicating the function inside _desktop_ and _shared_ folders, because it isn't possible to require a _shared/utils.js_ function from within a file that is not compiled through webpack. Is there a nicer alternative?

Reply:
That is fine for now! I added an issue #548 to add the `imports` support for the main process scripts.

But in addition of removing non ASCII characters we should also remove all special characters like `/ \ : * ? " < > |` and spaces. Or maybe it would be easier to just leave alpha numeric values and `-`?

Reply:
Agreed, alphanumeric with `-` will be easier.